src/Collapsible/index.jsx

```diff
@@ -24,6 +25,14 @@ class Collapsible extends React.Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
```

Comment:
@AlasdairSwan this change is to allow the credentials link to be able to open the collapsible